tests/scripts/components-build-system.sh

```diff
@@ -50,6 +50,8 @@ component_build_make_no_gen_files () {
     # GCC needs "as" in $PATH by default. To use GCC, we need to tell it where
     # to find the assembler. Or we can use clang which just works.
     CC="$(command -v clang)"
+    # For cleaning.
+    RM="$(command -v rm)"
```

Comment:
Minor: this variable is only used in the next commit.

Reply:
Do you want me to rewrite the history, since you're the only one who cares about the history so far?